.github/workflows/vendor-node-modules.yml

```diff
@@ -38,31 +27,14 @@ jobs:
         uses: ./setup-commit-signing/
         with:
           signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
```

Comment:
Probably best to move this after `npm install` to stop anything from retrieving this (as long as `setup-commit-signing` remains shell only with no JS).